Troubleshooting.md

```diff
@@ -96,14 +96,19 @@ modified launch arguments, like so:
   self.initialProps = @{};
 
   // some more code ...
-    
+
   NSMutableDictionary * launchOptionsWithURL = [self getLaunchOptionsWithURL:launchOptions];
 
   return [super application:application didFinishLaunchingWithOptions:launchOptionsWithURL];
 }
 
 ```
 
-This implementation is included in the example app in this repo and can be used for testing. 
+This implementation is included in the example app in this repo and can be used for testing.
 Make sure to use the URL scheme (rntest://) of the example app when testing.
 
+### iOS APNS Token Troubleshooting
+There is an open issue with [`@react-native-firebase/messaging`](https://github.com/invertase/react-native-firebase/issues/8022) where the SDK will uppercase any APNS token returned using `messaging().getAPNSToken()`.
+You can verify this by adding a log the `AppDelegate.m` file that prints the deviceToken (you will need to convert to a hex string).
+APNS Services is case-insensitive, so be sure you are not registering the same device token twice.
```

Comment:
Hmm, I'm not sure about this line. I've in the past not able to find official documentation to support what Apple thinks about push tokens. Also, there are some issues with getting a push synchronously after setting one so there may be some issues with asking developers to validate this. We handle this on our end so to keep integration light. My suggestion would be to not specify this but have this troubleshooting more of a FYI than anything actionable by the developer.